mod1/README.md

```diff
@@ -239,7 +247,7 @@ This is the part where the compiler screams at you for missing a semicolon (I ho
 
 I would highly recommend using additional warning flags. It might seem counter intuitive to purposefully ask the compiler to give you more errors, but it can really help point out when you've screwed up. Here's my personal stash (straight from a `Makefile`), you should look up what they do:
 
-`WARNINGS=-pedantic -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Winit-self -Wuninitialized -Wmissing-declarations -Werror=format-security`
+`WARNINGS=-pedantic -Wall -Wextra -Wmissing-prototypes -Wstrict-prototypes -Wmissing-declarations -Werror=format-security`
```

Comment:
Are these removed since they're included in other flags?

Reply:
yeah, they are included within -Wall (https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html)

Reply:
It looks like init-self is gcc specific. I made sure clang behaves the same way. 👍